content/01.abstract.md

```diff
@@ -5,3 +5,5 @@ Manubot applies the git-based software workflow to scholarly writing, enabling e
 
 This manuscript is a Manubot demo, intended to give users a playground to practice using Manubot.
 Everyone is encouraged to try writing with Manubot by editing this manuscript.
+
+Manubot is described in the paper titled "Open collaborative writing with Manubot".
```

Comment:
Please add a URL citation to this paper.

Reply:
That's a good idea!

Reply:
That's a good idea!

Reply:
I've added the citation and the manuscript is ready for re-review.